app/models/area_vet.rb

```diff
@@ -0,0 +1,2 @@
+class AreaVet < ApplicationRecord
```

Comment:
Wrong naming in my opinion. `Area` fullfills everything. We alreay know it is a veterinary